repositories/ValidationRepository.js

```diff
@@ -33,8 +39,56 @@ const updateValidationRule = input => {
   }).then(head);
 };
 
+const getPreviousAnswersForValidation = id =>
+  db("Answers")
```

Comment:
Should all these linked queries (not just here) be done in a transaction so we have a consistent view of the data? I appreciate this is probably an edge case where people are updating things as the queries are running.

Reply:
Yes but unnecessary at the moment as this could happen in multiple places in the application and there is no requirement to optimise for this scenario.  Based on our current UI you wouldn't be able to update an answer whilst doing this.

Reply:
As discussed this not a problem. We need to resolve here as we need locking etc to handle multiple users editing questionnaires.